flang/lib/Lower/OpenMP.cpp

```diff
@@ -17,6 +17,7 @@
 #include "flang/Parser/parse-tree.h"
 #include "mlir/Dialect/OpenMP/OpenMPDialect.h"
 #include "llvm/Frontend/OpenMP/OMPConstants.h"
+#include "llvm/Support/CommandLine.h"
```

Comment:
Are you planning to add a command-line option at some point?

Reply:
I was planning for version specific's later. This header is also useful while debugging. `dbgs` based. Anyway I'll remove this since it's not relevant WRT this patch.

Reply:
Ok. (Command line options ought to move to the new pass-oriented MLIR registration system in general.)